posthog/hogql_queries/insights/test/test_trends_query_runner.py

```diff
@@ -0,0 +1,280 @@
+                                "2020-01-11T12:00:00Z",
+                                "2020-01-12T12:00:00Z",
+                                "2020-01-13T12:00:00Z",
+                                "2020-01-15T12:00:00Z",
+                                "2020-01-17T12:00:00Z",
+                                "2020-01-19T12:00:00Z",
+                            ],
+                        ),
+                        Series(
+                            event="$pageleave",
+                            timestamps=[
+                                "2020-01-11T12:00:00Z",
+                                "2020-01-12T12:00:00Z",
+                                "2020-01-13T12:00:00Z",
+                            ],
+                        ),
+                    ],
+                ),
+                SeriesTestData(
+                    distinct_id="p2",
+                    events=[
+                        Series(event="$pageview", timestamps=["2020-01-09T12:00:00Z", "2020-01-12T12:00:00Z"]),
+                        Series(
+                            event="$pageleave",
+                            timestamps=[
+                                "2020-01-13T12:00:00Z",
+                            ],
+                        ),
+                    ],
+                ),
+                SeriesTestData(
+                    distinct_id="p3",
+                    events=[
+                        Series(event="$pageview", timestamps=["2020-01-12T12:00:00Z"]),
+                        Series(event="$pageleave", timestamps=["2020-01-13T12:00:00Z"]),
+                    ],
+                ),
+                SeriesTestData(
+                    distinct_id="p4",
+                    events=[
+                        Series(event="$pageview", timestamps=["2020-01-15T12:00:00Z"]),
+                        Series(event="$pageleave", timestamps=["2020-01-16T12:00:00Z"]),
+                    ],
+                ),
+            ]
+        )
+
+    def _create_query_runner(self, date_from, date_to, interval, series, trends_filters) -> TrendsQueryRunner:
+        query_series = [EventsNode(event="$pageview")] if series is None else series
+        query = TrendsQuery(
+            dateRange=DateRange(date_from=date_from, date_to=date_to),
+            interval=interval,
+            series=query_series,
+            trendsFilter=trends_filters,
+        )
+        return TrendsQueryRunner(team=self.team, query=query)
+
+    def _run_trends_query(
+        self, date_from, date_to, interval, series=None, trends_filters: Optional[TrendsFilter] = None
+    ):
+        return self._create_query_runner(date_from, date_to, interval, series, trends_filters).calculate()
+
+    def test_trends_query_label(self):
+        self._create_test_events()
+
+        response = self._run_trends_query(self.default_date_from, self.default_date_to, IntervalType.day)
+
+        self.assertEqual("$pageview", response.result[0]["label"])
```

Comment:
With pytest `assert x = y` is a great experience and has messages more useful than unittest's `assertEqual`, so let's lean towards it in new tests

Reply:
Ahh, nice, okay - will ensure to do that. Are there any reasons why we should use the base unittest functions over the inbuilt `assert` ?

Reply:
There's only one case I remember that the `assert ...` style doesn't support, and it's `self.assertDictContainsSubset()`. Otherwise I _think_ there are no reasons